### PR TITLE
fix: remove visible page divider

### LINKED ISSUE #1343: bug: Remove visibility page divider
This change introduce another visual bug. The <hr />  divider was left on the page. 

Screen Shot 2023-07-10 at 4 20 21 PM

_Originally posted by @bdougie in https://github.com/open-sauced/insights/issues/1331#issuecomment-1629862048_
            
## Solution

Remove the lower divider

### PR DESCRIPTION
**Pull request by LadegbayeStella**
## Description

This pull request removes the visible page divider on the add Insights page. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

 Fixes #1343 

## Mobile & Desktop Screenshots/Recordings
 **BEFORE**

image

**AFTER**

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?

No

### CODE REVIEW

**Review by babblebey — APPROVED** [review]
LGTM 🍕

Thanks for your contribution @LadegbayeStella.

Welcome to the OpenSauced🍕 community.

**Comment by bdougie** [comment]
Thanks for working on this @LadegbayeStella. Could you also remove the extra padding in this section.
Screen Shot 2023-07-17 at 4 49 38 PM

File: components/organisms/SuggestedRepoList/suggested-repo-list.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const SuggestedRepositoriesList = ({ reposData, onAddRepo }: SuggestedRepositori
         Suggested Repositories:
       </Title>
 
-      <div className="flex flex-col gap-3 pb-10 mt-6 border-b">
+      <div className="flex flex-col gap-3 pb-10 mt-6">
```
Comment (NsdHSO) [inline components/organisms/SuggestedRepoList/suggested-repo-list.tsx:18]:
```suggestion
      <div className="flex flex-col gap-3 mt-6">
```

File: components/organisms/SuggestedRepoList/suggested-repo-list.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const SuggestedRepositoriesList = ({ reposData, onAddRepo }: SuggestedRepositori
         Suggested Repositories:
       </Title>
 
-      <div className="flex flex-col gap-3 pb-10 mt-6 border-b">
+      <div className="flex flex-col gap-3 mt-6">
```
Comment (NsdHSO) [inline components/organisms/SuggestedRepoList/suggested-repo-list.tsx:18]:
```suggestion
      <div className="flex flex-col gap-3 pb-10 mt-6 border-b">
```

I think we should go with another approach.  [here](https://github.com/open-sauced/insights/blob/298718f1e7c5bd28df93e2e8e612e25a8c6dd097/components/organisms/Footer/footer.tsx#L63C1-L63C1) we should remove border-top. @bdougie  it is good like that 
image

**Comment by OgDev-01** [comment]
I think the border in the footer is ok as is. 

>Could you also remove the extra padding in this section.

I agree with this as the border in the footer serves as a visible divisor across all pages

**Pull request closed without merging**